src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/conversion.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/conversion.hpp"
+#include "shared_test_classes/single_layer/conversion.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace Conversion {
+
+INSTANTIATE_TEST_SUITE_P(smoke_ConvertCPULayerTest_7D_Dynamic, ConvertCPULayerTest,
```

Comment:
Why no to make these instances common? I expect it should work correctly on any target.

Reply:
These 7D tests go to the x86 reference implementation.